gapic-generator/templates/default/service/rest/service_stub/method/def/_response.erb

```diff
@@ -1,16 +1,21 @@
 <%- assert_locals method -%>
 
-uri, <%= method.rest.body_var_name %>, <%= method.rest.query_string_params_var_name %> = <%= method.rest.transcoding_helper_name %> request_pb
-response = @client_stub.make_<%= method.rest.verb %>_request(
+verb, uri, query_string_params, body = <%= method.rest.transcoding_helper_name %> request_pb
+query_string_params = if query_string_params.any?
+  query_string_params.map{ |p| p.split("=", 2) }.to_h
```

Comment:
See below how the final generated code (a few files down) indents this. Best to indent it that way in the template too, rather than depending on rubocop to fix it.

Reply:
:+1: